### PR TITLE
rename completeness to total score for greater clarity

### PR DESCRIPTION
**Pull request by bbrewer-papa-x**
smartcitiesdata/react_discovery_ui#13

co-authored-by: Kenny Drobnack <kdrobnack@pillartechnology.com>

### CODE REVIEW

File: lib/discovery_api/data/persistence.ex [hunk 1]
```diff
@@ -57,7 +57,7 @@ defmodule DiscoveryApi.Data.Persistence do
   defp decode_if_json(nil), do: nil
 
   defp decode_if_json(value) do
-    case Jason.decode(value) do
+    case Jason.decode(value, keys: :atoms) do
```
Comment (jeffgrunewald) [inline lib/discovery_api/data/persistence.ex]: Is this based on the structure of the individual datasets or generic to all datasets? Is this going to open up a DDoS of our atom table?

Comment (bbrewer-papa-x) [reply lib/discovery_api/data/persistence.ex]: Hmm, on reflection, there are user defined column names in here so we should play it safe

File: lib/discovery_api/stats/stats_calculator.ex [hunk 2]
```diff
@@ -56,9 +56,9 @@ defmodule DiscoveryApi.Stats.StatsCalculator do
     |> Prestige.execute(rows_as_maps: true)
   end
 
-  defp add_completeness_total(dataset_stats) do
+  defp add_total_score(dataset_stats) do
     score = CompletenessTotals.calculate_dataset_total(dataset_stats)
-    Map.put(dataset_stats, :completeness, score)
+    Map.put(dataset_stats, :total_score, score)
```
Comment (ManApart) [inline lib/discovery_api/stats/stats_calculator.ex]: Will this need to re-run the calculation before it shows up properly in the UI?

Comment (bbrewer-papa-x) [reply lib/discovery_api/stats/stats_calculator.ex]: Correct

Comment (bbrewer-papa-x) [reply lib/discovery_api/stats/stats_calculator.ex]: Thankfully, the UI will not be any more or less broken than it already is during this process

Comment (ManApart) [reply lib/discovery_api/stats/stats_calculator.ex]: That's fair. That's fair.

**Pull request merged**